doc/installing.rst

```diff
@@ -138,6 +138,22 @@ pandas) installed first. Then, install xarray with pip::
 
     $ pip install xarray
 
+We also maintain other dependency sets for different subsets of functionality::
+
+    $ pip install "xarray[io]"        # Install optional dependencies for handling I/O
+    $ pip install "xarray[accel]"     # Install optional dependencies for accelerating xarray
+    $ pip install "xarray[parallel]"  # Install optional dependencies for dask arrays
+    $ pip install "xarray[viz]"       # Install optional dependencies for visualization
+    $ pip install "xarray[complete]"  # Install all the above
+
+The above commands should install most of the `optional dependencies`_. However,
+some packages which are either not listed on PyPI or require extra
+installation steps are omitted. To know which dependencies would be installed,
+take a look at the ``[options.extras_require]`` section in setup.cfg_.
```

Comment:
I don't think we should link to the `stable` docs because this may be out-of-date for other branches. Maybe we can use the `literalinclude` directive?

Reply:
Great suggestion. Done.